### ISSUE TITLE
Reported conflict with  iThemes Security Pro 

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#46]
We received a report on a website running iThemes Security Pro that Accessibility Checker was unable to identify issues. They ended up deactivating iThemes Security Pro and scanning worked, but we should assess this and see if there's a way to make the two plugins work with one another.

More details in this [support ticket](https://equalizedigital.zendesk.com/agent/tickets/5640).

**Comment by SteveJonesDev** [comment]
This may be a settings issue. Under System Tweaks then try and disable the Disable PHP in Plugins checkbox, if it is currently enabled. I’ll try to find a copy of the plug-in to test and confirm.

**Comment by SteveJonesDev** [comment]
Timothy Jacobs from iThemes will provide a copy of iThemes Security Pro.

**Comment by SteveJonesDev** [comment]
In testing iThemes Security Pro I'm not having any issue scanning with Accessibility Checker. In the support ticket linked above it stated that the Accessibility Checker plugin is running on multiple sites on the same server setup successfully with iThemes Security Pro active. It's just one site that is having an issue. If we could get an export of the exact iThemes Security Pro setting they have set we could test those. I'm assuming it maybe another issues such as password protections or issue with database tables not being created.

**Comment by amberhinds** [comment]
Can you follow up with that person and see if they're still having issues?

I'll ask Nathan. I wonder if it was actually a different plugin conflict and we just suspected that it was iThemes Security.

**Comment by SteveJonesDev** [comment]
Closing this issue due to not being able to replicate the issue and the user is no longer using iThemes security.
